### PR TITLE
add velero and minio examples

### PR DESCRIPTION
**Pull request by nabuskey**
Adds an example of setting up velero and minio locally for backing up objects.

### CODE REVIEW

File: examples/local-backup/README.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+## Using it
+
+Once minio and velero ArgoCD applications are ready, you can start playing with it.
+
+MinIO console is accessible at [https://minio.cnoe.localtest.me:8443/login](https://minio.cnoe.localtest.me:8443/login)
+
+You can log in to the console by obtaining credentials:
+
+```bash
+kubectl -n minio get secret minio -o go-template='{{ range $key, $value := .data }}{{ printf "%s: %s\n" $key ($value | base64decode) }}{{ end }}'
```
Comment (nimakaviani) [inline examples/local-backup/README.md:66]: the secret name for my minio instance is `root-creds`. is that the default? show the above name be fixed?

File: examples/local-backup/demo/backup.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# velero backup create nginx-backup --selector app=nginx
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: nginx-backup
+  namespace: velero
+spec:
+  includedNamespaces:
+    - '*'
```
Comment (nimakaviani) [inline examples/local-backup/demo/backup.yaml:9]: probably a good idea to have `nginx-example` listed as the namespace

File: examples/local-backup/demo/restore.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# /velero restore create --from-backup nginx-backup
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: nginx-backup
+  namespace: velero
+spec:
+  backupName: nginx-backup
+  includedNamespaces:
+    - '*'
```
Comment (nimakaviani) [inline examples/local-backup/demo/restore.yaml:10]: same as above

File: examples/local-backup/README.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+This command:
+1. Creates a standard idpbuilder installation, a kind cluster and core packages (ArgoCD, Gitea, and Ingress-Nginx).
+2. Creates two custom packages: [MinIO](./minio.yaml) and [Velero](./velero.yaml).
+
+Once the command exits, you can check the status of installation by going to https://argocd.cnoe.localtest.me:8443/applications.
+
+Username is `admin`, and password is obtained with:
+```bash
+kubectl -n argocd get secret argocd-initial-admin-secret -o go-template='{{ range $key, $value := .data }}{{ printf "%s: %s\n" $key ($value | base64decode) }}{{ end }}'
+```
```
Comment (nimakaviani) [inline examples/local-backup/README.md:49]: This bit is not required. it gets printed every time someone runs the idpBuilder

**Review by nimakaviani — CHANGES_REQUESTED** [review]

**Comment by nimakaviani** [comment]
great stuff overall. left some minor comments!

**Comment by nabuskey** [comment]
Addressed comments. Please review when you have time

**Review by nimakaviani — APPROVED** [review]
LGTM, thanks!

**Pull request merged**